cytotable/utils.py

```diff
@@ -9,7 +9,7 @@
 from typing import Any, Dict, Union, cast
 
 import duckdb
-import pyarrow as pa
+import parsl
```

Comment:
Just confirming that you no longer wish to import `pyarrow`

Reply:
Thanks for the double check here. I noticed this dependency had been specified deeper in this module as I made the edits in this PR and decided to clean it up. Specifically, `_sqlite_mixed_type_query_to_parquet` has this dependency, but the import is now specified at a function-level in abidance with the style of Parsl apps (where the function is referenced).